flask_wtf/csrf.py

```diff
@@ -138,30 +138,16 @@ def __init__(self, app=None):
             self.init_app(app)
 
     def init_app(self, app):
+        self._app = app
         app.jinja_env.globals['csrf_token'] = generate_csrf
         app.config.setdefault(
             'WTF_CSRF_HEADERS', ['X-CSRFToken', 'X-CSRF-Token']
         )
         app.config.setdefault('WTF_CSRF_SSL_STRICT', True)
         app.config.setdefault('WTF_CSRF_ENABLED', True)
+        app.config.setdefault('WTF_CSRF_CHECK_DEFAULT', True)
```

Comment:
I hesitated too. Setting `WTF_CSRF_ENABLED` to `False` while still calling csrf.protect() in `before_request` sounds confusing, since CSRF is not technically disabled, Flask-WTF is just not checking every view by default automatically. So I thought a "check-by-default" setting would make more sense.

Still, I can remove it and use `WTF_CSRF_ENABLED` if you'd like.